### PR TITLE
Sales eng quick clear

### PR DESCRIPTION
**Pull request by alexgraham-da**
- all clearing to quick setup
- add clearing role accept
- add allocation account request
- add account request accept

### CODE REVIEW

File: ui2/src/pages/custody/New.tsx [hunk 1]
```diff
@@ -48,19 +70,40 @@ const NewComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> =
...
+        if (!nomineeIdentity) return;
+        const request: RequestOpenAllocationAccount = {
+          accountId: {
+            signatories: {
+              textMap: { [service.payload.provider]: {}, [service.payload.customer]: {} },
+            },
+            label: accountName,
+            version: '0',
+          },
+          // observers: { textMap: { publicParty } },
```
Comment (Sarahb-da) [inline ui2/src/pages/custody/New.tsx:100]: `[publicParty]` ?

File: ui2/src/pages/custody/New.tsx [hunk 2]
```diff
@@ -48,19 +70,40 @@ const NewComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> =
...
+            version: '0',
+          },
+          observers: [publicParty],
+          ctrls: [service.payload.provider, service.payload.customer],
+        };
+        await ledger.exercise(Service.RequestOpenAccount, service.contractId, accountRequest);
+        history.push('/app/custody/requests');
+      case AccountType.ALLOCATION:
+        const nomineeIdentity = identities.find(i => i.payload.customer === accountNominee);
+        console.log('going...');
```
Comment (Sarahb-da) [inline ui2/src/pages/custody/New.tsx:90]: test log  :)

File: ui2/src/pages/login/QuickSetup.tsx [hunk 3]
```diff
@@ -60,9 +64,16 @@ import { useAutomations, AutomationProvider } from '../../context/AutomationCont
...
 import { Service } from '@daml.js/da-marketplace/lib/Marketplace/Custody/Service';
 
-type Offer = CustodianOffer | DistributorOffer | SettlementOffer | ExchangeOffer | MatchingOffer;
+type Offer =
+  | ClearingOffer
+  | CustodianOffer
+  | DistributorOffer
+  | SettlementOffer
+  | ExchangeOffer
+  | MatchingOffer;
```
Comment (Sarahb-da) [inline ui2/src/pages/login/QuickSetup.tsx:73]: oo i forgot to add `RegulatorOffer` to this - if u commit again pls add!

**Review by Sarahb-da — APPROVED** [review]

File: ui2/src/pages/custody/New.tsx (hunk 1 above)
Comment (alexgraham-da) [reply ui2/src/pages/custody/New.tsx:100]: Ah...was a weird type issue with `Set`. I decided to not include the public party as an observer anyway here.

**Pull request merged**